### PR TITLE
Add useFollow composable and follow button

### PR DESCRIPTION
**Pull request by samuveth**

### CODE REVIEW

**Comment by samuveth** [comment]
[Add follow action](https://trello.com/c/V0CDf1aC/228-add-follow-action)

File: src/helpers/EIP712.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import Client from '@snapshot-labs/snapshot.js/src/sign';
+
+const hubUrl = process.env.VUE_APP_HUB_URL || 'https://testnet.snapshot.org';
```
Comment (bonustrack) [inline src/helpers/EIP712.ts:3]:
```suggestion
const hubUrl = process.env.VITE_HUB_URL || 'https://testnet.snapshot.org';
```

**Comment by samuveth** [comment]
image
Need to look into this.

**Comment by samuveth** [comment]
We can use "Following" once we have the meta-action implemented, until then the Metamask popup is causing to many issues with the hover listeners.

**Comment by bonustrack** [comment]
>  the Metamask popup is causing to many issues with the hover listeners.
What do you mean?

**Comment by samuveth** [comment]
@bonustrack when you click "Unfollow" the Metamask popup will prevent the `@mouseleave` from triggering.

**Comment by bonustrack** [comment]
@samuveth Here is some review:
- Can you do the change on Snapshot.js to support both Wallet and Web3Provider?
- Can you change Follow / Unfollow to Join and Leave?

**Comment by samuveth** [comment]
Requires https://github.com/snapshot-labs/snapshot.js/pull/353

**Comment by samuveth** [comment]
Continued here https://github.com/snapshot-labs/snapshot/pull/560

**Pull request closed without merging**